### PR TITLE
[13.0][MIG] crm_claim_type: Migration to 13.0

### PR DESCRIPTION
**Pull request by mamcode**
This depends on #316

### CODE REVIEW

**Comment by mamcode** [comment]
@simahawk Fixed, please give me a review :)

File: crm_claim_type/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2015 Vauxoo: Yanina Aular <yani@vauxoo.com>,
+# Copyright 2015 Vauxoo: Osval Reyes <osval@vauxoo.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "CRM Claim Types",
+    "category": "Customer Relationship Management",
+    "summary": "Claim types for CRM",
+    "author": "Odoo Community Association (OCA)," "Vauxoo," "Ursa Information Systems",
```
Comment (simahawk) [inline crm_claim_type/__manifest__.py]: OCA should be last item

**Review by HaraldPanten — APPROVED** [review]
Functional review LGTM 👍

File: crm_claim_type/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+This modules do not require special configurations.
```
Comment (simahawk) [inline crm_claim_type/readme/CONFIGURE.rst]:
```suggestion
This modules does not require special configurations.
```

File: crm_claim_type/models/crm_claim_stage.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 URSA Information Systems
```
Comment (simahawk) [inline crm_claim_type/models/crm_claim_stage.py]: order inverted here?

File: crm_claim_type/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1,6 @@
+Includes a way to classify claims adding the concept of type and with this,
+it allows to configure claim stages depending on claim types.
+
+This module includes Customer, Supplier and Other claim types as default data
+and its own stages relation, and also makes few stages common for different
+claim types
```
Comment (simahawk) [inline crm_claim_type/readme/DESCRIPTION.rst]:
```suggestion
Provides a way to classify claims by type and to configure claim stages depending on claim types.

Includes Customer, Supplier and Other claim types by default,
as well as stages configuration.
```

File: crm_claim_type/readme/USAGE.rst [hunk 5]
```diff
@@ -0,0 +1,3 @@
+To use this module, you need to:
+
+* Go to Claims, and select a claim type when creating/editing a claim
```
Comment (simahawk) [inline crm_claim_type/readme/USAGE.rst]:
```suggestion
* Go to Claims and select a claim type when creating/editing a claim
```

**Review by simahawk — CHANGES_REQUESTED** [review]
LG, some small remarks.

File: crm_claim_type/models/crm_claim_stage.py (hunk 3 above)
Comment (mamcode) [reply crm_claim_type/models/crm_claim_stage.py]: These lines of copyright should be ordered by year ?

Comment (mamcode) [reply crm_claim_type/models/crm_claim_stage.py]: I have ordered the lines by year :)

File: crm_claim_type/__manifest__.py (hunk 1 above)
Comment (mamcode) [reply crm_claim_type/__manifest__.py]: Done

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by simahawk — APPROVED** [review]

**Pull request merged**